src/main/java/au/com/sealink/quicktravel/client/models/VehicleType.java

```diff
@@ -0,0 +1,251 @@
+package au.com.sealink.quicktravel.client.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class VehicleType {
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("minimum_length")
+    @Expose
+    private Object minimumLength;
+    @SerializedName("fixed_length")
+    @Expose
+    private Boolean fixedLength;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("description")
+    @Expose
+    private String description;
+    @SerializedName("default")
+    @Expose
+    private Boolean _default;
+    @SerializedName("trailer")
+    @Expose
+    private Boolean trailer;
+    @SerializedName("maximum_length")
+    @Expose
+    private Object maximumLength;
+    @SerializedName("position")
+    @Expose
+    private Integer position;
+    @SerializedName("fixed_weight")
+    @Expose
+    private Boolean fixedWeight;
+    @SerializedName("default_weight")
+    @Expose
+    private String defaultWeight;
+    @SerializedName("detail_prompt")
+    @Expose
+    private String detailPrompt;
+    @SerializedName("has_cargo")
+    @Expose
+    private Boolean hasCargo;
+    @SerializedName("code")
+    @Expose
+    private String code;
+    @SerializedName("appear_on_manifest")
+    @Expose
+    private Boolean appearOnManifest;
+    @SerializedName("created_at")
+    @Expose
+    private Date createdAt;
+    @SerializedName("updated_at")
+    @Expose
+    private Date updatedAt;
+    @SerializedName("multiply_length")
+    @Expose
+    private Boolean multiplyLength;
+    @SerializedName("default_length")
+    @Expose
+    private String defaultLength;
```

Comment:
String (shouldn't it be double?)

Reply:
interestingly enought the json from QuickTravel is a string :( hence the tool picked it up wrong.